src/components/ConfirmModal.js

```diff
@@ -87,4 +91,4 @@ const ConfirmModal = props => (
 ConfirmModal.propTypes = propTypes;
 ConfirmModal.defaultProps = defaultProps;
 ConfirmModal.displayName = 'ConfirmModal';
-export default withWindowDimensions(ConfirmModal);
+export default compose(withWindowDimensions, withLocalize)(ConfirmModal);
```

Comment:
Could you break this up into multiple lines like the rest of the composed exports, please?